other/ethhub-cftc-response.md

```diff
@@ -10,11 +10,16 @@ DEADLINE: February 9, 2019
 
 ### 1. What was the impetus for developing Ether and the Ethereum Network, especially relative to Bitcoin?
 
-It's first vitally important to distinguish between Ether and Ethereum. Ethereum is an open-source, blockchain-based distributed ledger with decentralized control that supports general computing and economic activities. Anyone is able to build and deploy decentralized products and services that run on top of Ethereum by creating highly flexible programmable transactions, called 'smart contracts'. This smart contract technology has proven to be attractive to developers since it enables them to create programs and business logic that run exactly as coded, trustlessly and with no down time.
```

Comment:
It is useful to distinguish between Ether and Ethereum right off the bat for those who do not know the difference. Mention of smart contracts for the laymen is useful as well. Maybe mention Turing completeness enables smart contracts.

Reply:
It’s not as important to define Ether vs Ethereum as the questions later lend to the impression that they already know this. It is important to include more information than this author wants to update this to. For example, the impetus is not just coopting Bitcoin tech and slapping on a quasi-turing complete programming language and execution environment. Bitcoin Script is not expressive or flexible enough to support many of the use cases Ethereum’s creators envisioned. Additionally, an unwillingness of the Bitcoin core client maintainers to enable opcodes and discuss block size increases made the Bitcoin blockchain a non-starter for those same use cases. This is all relevant and lends to the motivation for developing a separate network vs simply building on Bitcoin a la RSK.

Reply:
>For example, the impetus is not just coopting Bitcoin tech and slapping on a quasi-turing complete programming language and execution environment. Bitcoin Script is not expressive or flexible enough to support many of the use cases Ethereum’s creators envisioned. Additionally, an unwillingness of the Bitcoin core client maintainers to enable opcodes and discuss block size increases made the Bitcoin blockchain a non-starter for those same use cases. This is all relevant and lends to the motivation for developing a separate network vs simply building on Bitcoin a la RSK.

Great point. We need to keep in mind that they want to know the practical reason of why Ethereum came into existence.

Reply:
thanks for the input, my opinion here is that we need to prioritize changes in this document - incorrect information should be fixed before further explanations for the layman (even though they are important).
this is why i think my pull request should be merged, later anyone and myself will be able to add and contribute to a correct answer and make it better.